docs/userguide/packaging/index.md

```diff
@@ -25,7 +25,7 @@ with -DU_STATIC_IMPLEMENTATION. Also see [How To Use ICU](../howtouseicu.md).
 ### Reduce the number of libraries used
```

Comment:
Can't really comment at the top of the file, but maybe consider adding the Unicode copyright for the linter:
```
<!--
© 2020 and later: Unicode, Inc. and others.
License & terms of use: http://www.unicode.org/copyright.html
-->
```

Reply:
Yes, please add copyright lines at the top of each .md file.